src/openrct2/management/Award.cpp

```diff
@@ -552,7 +552,10 @@ static bool award_is_deserved_best_custom_designed_rides(sint32 awardType, sint3
 }
 
 /** At least 5 colourful rides and more than half of the rides are colourful. */
-static const uint8 dazzling_ride_colours[] = {5, 14, 20, 30};
+static const uint8 DazzlingRideColours[] =
+{
+    COLOUR_BRIGHT_PURPLE, COLOUR_BRIGHT_GREEN, COLOUR_LIGHT_ORANGE, COLOUR_BRIGHT_PINK,
```

Comment:
These changes do not really belong here, but you can leave them as they are, AFAIC.

Reply:
I just noticed after pushing, one moment :P

Reply:
I was working on those in another branch.